account_invoice_margin/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Account Invoice Margin",
     "summary": "Show margin in invoices",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.0.1",
```

Comment:
for your information, such change is useless 
when merging with @OCA-git-bot, the bump of the version will be done. (if asked by maintainer).

Reply:
I got it, thanks